vite.config.ts

```diff
@@ -3,6 +3,14 @@ import react from '@vitejs/plugin-react';
 import dts from 'vite-plugin-dts';
 import packageJson from './package.json';
 
+const makeExternalPredicate = (externals: string[]) => {
+  if (externals.length === 0) {
+    return () => false;
+  }
+  const pattern = new RegExp(`^(${externals.join('|')})($|/)`);
+  return (id: string) => pattern.test(id);
+};
```

Comment:
Alternatively could be something like:
```suggestion
const externalsPredicate = (id: string) => !id.startsWith('.')
```